fe/src/pages/admin/category/ListCategory.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { getAllAction } from "../../../redux/actions";
+
+export default function ListCategory() {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const dispatch = useDispatch();
+  const categories = useSelector((state) => state.categories[0]);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  useEffect(() => {
+    dispatch(getAllAction("api/lessons"));
+  }, []);
+
+  if (!categories) {
+    return (
+      <div className="card d-flex justify-content-center">
+        <h1 className="mx-auto">No Categories Available</h1>
+      </div>
+    );
+  }
+
+  return (
+    <Fragment>
+      {categories && (
```

Comment:
- remove this condition. you can return the following without the condition since nag check kana sa itaas.